analysis_runner/dataproc.py

```diff
@@ -21,6 +22,102 @@
 PYFILES_ZIP = 'pyfiles.zip'
 
 
+class DataprocCluster:
+    """
+    Helper class that represents a Dataproc cluster created within a Batch
+    """
+
+    def __init__(
+        self,
+        batch: hb.Batch,
+        start_job: hb.batch.job.Job,
+        stop_job: hb.batch.job.Job,
+        cluster_name: str,
+    ):
+        self.batch = batch
+        self.start_job = start_job
+        self.stop_job = stop_job
+        self.cluster_name = cluster_name
+
+    def add_job(
```

Comment:
Should this return the job object, so one can define dependencies between different jobs that are submitted to the same cluster?

Reply:
Good catch. Done!